monitor/services/pom.xml

```diff
@@ -200,5 +200,25 @@
 			<version>2.2.4</version>
 			<scope>provided</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.4</version>
```

Comment:
Let's have this in the parent POM under <dependencyManagement> ... I think we would like to move to commons-lang3 in general so having it there makes it easier to manage and avoid version conflicts.